charts/agent/tests/role_test.yaml

```diff
@@ -0,0 +1,39 @@
+suite: Agent Role Tests
+templates:
+  - templates/role.yaml
+  - templates/rolebinding.yaml
+tests:
+  - it: Role and RoleBinding created when rbac.create is set
+    set:
+      rbac:
+        create: true
+    asserts:
+      - containsDocument:
+          name: RELEASE-NAME-agent
+          apiVersion: rbac.authorization.k8s.io/v1
+          kind: Role
+        template: templates/role.yaml
+      - containsDocument:
+          name:  RELEASE-NAME-agent
+          apiVersion: rbac.authorization.k8s.io/v1
+          kind: RoleBinding
+        template: templates/rolebinding.yaml
+
+  - it: RoleBinding linked with Role and ServiceAccount
+    set:
+      rbac:
+        create: true
```

Comment:
Same as before

Reply:
Hey @mavimo - The test you suggested above will ensure that the Role and RoleBinding are not created. This test is validating that when the RoleBinding is created the content is what is expected. If we have already validated that the  RoleBinding is not created, I'm not sure what the negative test is in this scenario. Am I missing something?

Reply:
Right, sorry, we need to test "don't create role & role binding" when `rbac.create = false`

Reply:
Are we okay to merge this PR then?

Reply:
I add the test for `rbac.create = false` in [d30ccee39811215b1bfc0cd6739fdeaa9e6859f6](https://github.com/sysdiglabs/charts/pull/1041/files#diff-f292da8564fc3a39cb5aaf5a4f2d61861163037583f1646aaaf3d47d49318016R41-R51)